src/System.Net.Primitives/src/System.Net.Primitives.csproj

```diff
@@ -17,6 +17,7 @@
     <!-- SYSTEM_NET_PRIMITIVES_DLL is required to allow source-level code sharing for types defined within the
          System.Net.Internals namespace. -->
     <DefineConstants>$(DefineConstants);SYSTEM_NET_PRIMITIVES_DLL</DefineConstants>
+    <DefineConstants Condition="'$(TargetGroup)' == 'uapaot'">$(DefineConstants);uapaot</DefineConstants>
```

Comment:
No that isn't the correct condition. You should use TargetGroup in the condition but you need to decide if you need a new unique uapaot build or can this be made public for both uap and uapaot? If you need a new unique uapaot build you need to add that to the configurations.props file but if you can have this public for both just update the condition to be based on uap.